File: backend/src/Modules/Modules.Content.Presentation/Articles/Create/CreateArticleRequest.cs
namespace Modules.Content.Presentation.Articles.Create;

internal sealed record CreateArticleRequest(
    string Slug,
    string TitleHu,
    string? TitleEn,
    string? SummaryHu,
    string? SummaryEn,
    string ContentHu,
    string? ContentEn,
    int DisplayOrder);

File: frontend/henn-laszlo/projects/content-data-access/src/lib/models/create-article-request.ts
export interface CreateArticleRequest {
  readonly slug: string;
  readonly titleHu: string;
  readonly titleEn: string | null;
  readonly summaryHu: string | null;
  readonly summaryEn: string | null;
  readonly contentHu: string;
  readonly contentEn: string | null;
  readonly displayOrder: number;
}